web/ViewModels/AddressModel.cs

```diff
@@ -23,4 +24,25 @@ public class AddressModel
         @"([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))\s?[0-9][A-Za-z]{2})",
         ErrorMessage = "Please enter a valid postcode")]
     public string? Postcode { get; set; }
+
+    public string ToShortAddressString()
+    {
+        var address = new StringBuilder(AddressLine1);
+        if (AddressLine2 != null)
+        {
+            address.Append($", {AddressLine2}");
+        }
+
+        if (AddressLine3 != null)
+        {
+            address.Append($", {AddressLine3}");
+        }
+
+        if (TownOrCity != null)
+        {
+            address.Append($", {TownOrCity}");
+        }
+
+        return address.ToString();

```

Comment:
This could be simplified down a bit, we could use ternaries and do something like
```suggestion
        var address = AddressLine1;
        address += AddressLine2 != null ? $", {AddressLine2}" : "";
        address += AddressLine3 != null ? $", {AddressLine3}" : "";
        address += TownOrCity != null ? $", {TownOrCity}" : "";

        return address;
```